src/index.html

```diff
@@ -5,9 +5,160 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="shortcut icon" type="image/png" href="./images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,400i,500,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <img class="header__logo" src="./images/logo.png" alt="company logo">
+        <p class="header__phone">+1 234 555-55-55</p>
+        <a class="header__call-link" href="#">call to order</a>
+      </div class>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
```

Comment:
```suggestion
          <li class="nav__item">
```
будет выглядеть лучше